src/clear/model/modules/has_timestamps.cr

```diff
@@ -24,5 +24,13 @@ module Clear::Model::HasTimestamps
       model.updated_at = Time.local if model.changed? && !model.updated_at_column.changed?
     end
 
+    # Saves the record with the updated_at set to the current time.
+    def touch
+      now = Time.local
```

Comment:
Few improvement needed:

- Make the function pure by adding a default parameter now: `def touch(now = Time.local)`
- self.save return `true|false`. I think returning `self` might be better. It would allow function chaining like `model.touch.get_some_juicy_stuff`

Reply:
Usage of `self.save!` instead of `self.save` to avoid silent fail (`save!`) throw an error in case it cannot be saved.